runtime/src/gpu/amd/gpu-amd.c

```diff
@@ -378,4 +378,8 @@ bool chpl_gpu_impl_can_reduce(void) {
   return ROCM_VERSION_MAJOR>=5;
 }
 
+bool chpl_gpu_impl_can_cub_sort(void){
+  return chpl_gpu_impl_can_reduce();
```

Comment:
`can_sort`

Reply:
I think `chpl_gpu_impl_can_sort` and `chpl_gpu_can_sort` make it sound like it checks if our in house sort can sort it as well. 
I would want to make a distinction between the two. I suggested `chpl_gpu_impl_can_extern_sort` above, `chpl_gpu_impl_can_runtime_sort` is also an option

Reply:
We can have the name discussion in the thread above.

Sorry about posting an unhelpful comment. This line calls `chpl_gpu_impl_can_reduce`. It should call the `_sort` version.

Reply:
I've made it call `chpl_gpu_impl_can_reduce` on purpose because both of them have the same check needed: `ROCM_VERSION_MAJOR>=5`. So instead of duplicating the check, I just call the function above it